examples/hybrid/src/main.rs

```diff
@@ -0,0 +1,53 @@
+#![feature(let_chains)]
+
+mod grpc;
+mod web;
+
+mod hybrid;
+
+use std::{convert::Infallible, net::SocketAddr};
+
+use axum::response::Response;
+use hyper::{server::conn::AddrStream, service::make_service_fn};
+use tower::ServiceExt;
+use tracing::info;
+use tracing_subscriber::FmtSubscriber;
+
+pub type BoxError = Box<dyn std::error::Error + Send + Sync>;
+pub type Result<T> = std::result::Result<T, BoxError>;
+
+fn make_hybrid_service<Web, Grpc>(web: Web, grpc: Grpc) -> hybrid::HybridService<Web, Grpc> {
+    hybrid::HybridService::new(web, grpc)
+}
```

Comment:
Not sure we need a function that just calls `hybrid::HybridService::new`. Can just call `hybrid::HybridService::new` directly.